### ISSUE TITLE
DHCP (0.0.0.0) alternative for IPv6 

### ISSUE THREAD

**Issue by 3deep5me** [kube-vip/kube-vip#632]
**Is your feature request related to a problem? Please describe.**
I'm always frustrated when i want to public expose a LoadBalancer service with kube-vip within a (home)-network where i do not have a public ipv4. I sadly have not the possibility to configure a ipv6-subnet over the kube-vip ccm because i have a dynamic ipv6 prefix. 

**Describe the solution you'd like**
Similar to `0.0.0.0` on ipv4 side i want to set `'::'` on the `spec.loadBalancerIP` field.
When kube-vip sees a Service on this address, it should create a macvlan interface on the host and request/create a IPv6 address over DHCPv6. Once this address is provided, it should assign it as the LoadBalancer IP and update the Kubernetes Service. (Same as in the ipv4 dhcp feature)
[dhcp-feature](https://kube-vip.io/docs/usage/kubernetes-services/?query=dhcp#using-dhcp-for-load-balancers-experimental-kube-vip-v021).

**Describe alternatives you've considered**
I think this could be also accomplished with SLACC but SLACC as far as i know less common.
Automated change of the ccm config-map. But i think its really unhandy.

**Additional context**
This feature would perfectly integrate into the [cluster-api-provider-proxmox (CAPPX)](https://github.com/sp-yduck/cluster-api-provider-proxmox/tree/main#cluster-api-provider-proxmox-cappx) in a ipv6 home-lab setup which would bring easy HA K8s-deployment to Proxmox a huge step forward.

**Comment by lz37** [comment]
+1, by the way. i perfer slaac

**Comment by ChandanTeekinavar** [comment]
@3deep5me Did you got any solution for IPv6?

**Comment by thebsdbox** [comment]
I don't think k this would be a huge change. We could also accomplish this with a service created with an annotation potentially.

**Comment by 3deep5me** [comment]
@ChandanTeekinavar nope, did not work in this direction, my home-lab is currently only accessible over VPN.
@thebsdbox Just saw the 1.0 release - congratz - love to see the project growing/maturing! 
I just looked at the DHCP code, doing more or less the same for SLACC looks hard for me, as a go newbie. 
But i would like to help by writing documentation or ci tests or something like this. I would love to see this feature. 

What also comes to my mind, my provider changes the prefix every 12h hours, so it would be great if this scenario would be covered.

**Comment by ChandanTeekinavar** [comment]
@thebsdbox I was trying to do the DHCPv6 allocation for the VIP, But I am not able to get the IPv6 address. I am giving the below annotation 
```kube-vip.io/loadbalancerIPs: "::"```
But I am getting as 
```2025/09/12 12:15:47 WARN could not add ip: cannot assign requested address                                                                                                                    
2025/09/12 12:15:47 INFO Broadcasting NDP update ip=:: hwaddr="\xbc$\x11\x9a\xa0+" interface=ens18                                                                                            
2025/09/12 12:15:50 INFO Broadcasting NDP update ip=fd00:1:1::123 hwaddr="\xbc$\x11\x9a\xa0+" interface=ens18                                                                                 
2025/09/12 12:15:50 WARN could not add ip: cannot assign requested address                                                                                                                    
2025/09/12 12:15:50 INFO Broadcasting NDP update ip=:: hwaddr="\xbc$\x11\x9a\xa0+" interface=ens18      
```
I am not getting how can I achive this in the IPv6 only cluster. I am using v1.0.0 version.

**Comment by Cellebyte** [comment]
Something is on the way #1356 .

**Comment by Cellebyte** [comment]
This should be fixed in the next nightly build.
